### PR TITLE
switched to EAPI=5, distutils-r1

### PR DESCRIPTION
**Pull request by noelbk**
Hi, thanks for g-pypi.  It was really useful when I had to generate a bunch of python ebuilds.  I upgraded it and also fixed some unit tests.

### CODE REVIEW

File: gpypi/templates/ebuild.jinja [hunk 1]
```diff
@@ -56,19 +55,12 @@ S="{{ s }}"
...
 {% endif %}
 
-{% if docs_dir or examples_dir %}
-src_install() {
-    distutils_src_install
 {% if docs_dir %}
-    if use doc; then
-        dodoc -r {{ docs_dir }}
-    fi
+DOCS={{ docs_dir }}
```
Comment (domenkozar) [inline gpypi/templates/ebuild.jinja:59]: shouldn't `DOCS` be optional?

Comment (domenkozar) [inline gpypi/templates/ebuild.jinja:59]: I see, it is, nevermind.

**Comment by domenkozar** [comment]
Looks good, thank you!

**Comment by noelbk** [comment]
Great.

I just ran into a problem making an ebuild for pycassa.  I ened up with
"test" in RDEPEND but not IUSE, and emerge choked.  Where's the best place
to fix that?

LICENSE="MIT"
KEYWORDS="~amd64"
SLOT="0"
IUSE=""  ### <-- WRONG, should be "test"

DEPEND="dev-python/setuptools"
RDEPEND="dev-python/setuptools
dev-python/thrift
test? ( dev-python/nose )"

## 

Noel

**Comment by domenkozar** [comment]
if package supports testing then test should be added to IUSE in the template template

**Comment by noelbk** [comment]
from ebuild.jinja:

IUSE="{{ use|join(" ") }}"
RDEPEND="{{ rdepend|list|sort|join("\n\t") }}"

So. IUSE and RDEPEND come from sets.  just make sure that when test gets
into rdepend it gets into IUSE too.

**Pull request merged**